public/app/plugins/datasource/prometheus/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -15,24 +15,36 @@ const options: monacoTypes.editor.IStandaloneEditorConstructionOptions = {
   fixedOverflowWidgets: true,
   folding: false,
   fontSize: 14,
-  lineDecorationsWidth: 8,
+  lineDecorationsWidth: 8, // used as "padding-left"
```

Comment:
Instead of adding a comment here, maybe we could extract it to a const and use in both places?

Reply:
unfortunately these values are more "magical" :-( . for example, the padding-right value, all i can tell you that with it set to `8` it looks ok. but it also looks ok with `1`. (it probably has some builtin min-value-for-scrollbar-width .. maybe.. not sure). so for now i recommend just keeping them here.